python/fastquant/strategies/custom.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Import standard library
+# Strategy module for custom indicators input to backtest
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
+
+# Import modules
+import backtrader as bt
+
+# Import from package
+from fastquant.strategies.base import BaseStrategy
+from fastquant.indicators.custom import CustomIndicator
+
+
+class CustomStrategy(BaseStrategy):
+    """
+    Simple moving average crossover strategy
```

Comment:
We need details on the docstring to provide context and examples.

```suggestion
    implements a chosen dataframe column as a custom indicator (column name set as "custom" by default).
    
    The strategy is structured similar to RSIStrategy where you can set an upper_limit, above which the asset is sold (considered "overbought"), and a lower_limit, below which the asset is sold (considered "underbought). upper_limit is set to 95 by default, while lower_limit is set to 5 by default.
```